src/event.rs

```diff
@@ -496,5 +499,10 @@ impl<N: Notify> Processor<N> {
         self.key_bindings = config.key_bindings().to_vec();
         self.mouse_bindings = config.mouse_bindings().to_vec();
         self.mouse_config = config.mouse().to_owned();
+        self.selection_buffer = if config.selection().save_to_clipboard {
+            ::copypasta::Buffer::Primary
+        } else {
+            ::copypasta::Buffer::Selection
+        };
```

Comment:
Should this really be one or the other? Should selecting things with `save_to_clipboard` on also fill the selection buffer?

Reply:
I support the idea to still paste it into the selection clipboard.